src/settings.zig

```diff
@@ -189,6 +189,8 @@ pub fn save() void {
 pub const launchConfig = struct {
 	pub var cubyzDir: []const u8 = "";
 	pub var autoEnterWorld: []const u8 = "";
+	pub var headlessServer: bool = false;
+	pub var autoCreateWorldConfig: main.server.world_zig.WorldSettings = undefined;
```

Comment:
Could you also adjust the default launchConfig to show all these new fields please?